framework-cloud/framework-openshift-operator/src/main/java/org/kie/cloud/openshift/operator/scenario/builder/ClusteredWorkbenchKieServerDatabasePersistentScenarioBuilderImpl.java

```diff
@@ -68,11 +64,8 @@ public ClusteredWorkbenchKieServerDatabasePersistentScenarioBuilderImpl() {
         });
 
         CommonConfig commonConfig = new CommonConfig();
-        commonConfig.setAdminUser(DeploymentConstants.getWorkbenchUser());
```

Comment:
Is this correct? I think operators can still work as before (using all the users).

Reply:
I suppose that operator should work in both ways - using old users and new user.
Old users should be there for compatibility purposes?

Reply:
It seems the operators are now only using the new user
the only difference is that in the templates, the trial won't use the secret, but the plain provided users
this has been also confirmed with Ruben